[esx]/es_extended/server/main.lua

```diff
@@ -600,3 +601,18 @@ AddEventHandler('txAdmin:events:scheduledRestart', function(eventData)
 		end)
 	end
 end)
+
+RegisterNetEvent('esx:setDuty')
+AddEventHandler('esx:setDuty', function(state)
+    local xPlayer = ESX.GetPlayerFromId(source)
+    if xPlayer.job.onduty == state then return end
+    
+    if state then
```

Comment:
I would change that to `bool` to make cleaner. But doesn't really change anything.